tests/unit/remote/test_gdrive.py

```diff
@@ -18,7 +16,7 @@ class Repo(object):
     tmp_dir = ""
 
 
-class TestRemoteGDrive(TestCase):
+class TestRemoteGDrive(object):
```

Comment:
why this change? is it important for this PR?

Reply:
it was in original Ruslan's PR. Just left it there. Probably, it's not important, but seems it removes not mandatory dependency on `TestCase`.